react/src/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import {render} from 'react-dom';
+import {BrowserRouter, Route, Switch} from 'react-router-dom';
+import './index.css';
+import config from "./config";
+import Home from './components/home';
+
+render(
+      <BrowserRouter>
+        <Switch>
+          <Route render={(routeProps) =>
+              <Home {...routeProps} {...config}/>}/>
+        </Switch>
```

Comment:
Do we need the swtich? Could we nix the Route and just render the Home component here?

Reply:
Surely we can. I've added a Route so developers can easily add their own routes without much typing.
I'm fine with 2 cases - just let me know wht u prefer best